exporters/etw/include/opentelemetry/exporters/etw/etw_logger.h

```diff
@@ -163,7 +161,7 @@ class Logger : public opentelemetry::logs::Logger
         ActivityIdPtr = &ActivityId;
       }
     }
-    evt[ETW_FIELD_PAYLOAD_NAME]              = std::string(name.data(), name.length());
+    evt[ETW_FIELD_PAYLOAD_NAME]              = std::string(provId.data(), provId.length());
```

Comment:
Payload name should ideally be the property of each Log, and not the Logger. But I understand we don't have much option without `name` field.

Reply:
What's your suggestion here? I'm not familiar with ETW, and I don't know what should be in ETW data.

Reply:
If we are not changing the Logger API, do you think we can keep the `name` in the ETW.

Reply:
I think we can keep `name` temporary ,but we should remove `name` in opentelemetry-api eventually, or it will confuse users.